sdk/iot/Azure.Iot.Hub.Service/src/ModulesClient.cs

```diff
@@ -18,7 +18,7 @@ namespace Azure.Iot.Hub.Service
     public class ModulesClient
     {
         private const string ContinuationTokenHeader = "x-ms-continuation";
-        private const string HubModuleQuery = "select * from modules";
+        private const string HubModuleQuery = "select * from devices.modules";
```

Comment:
The service expected this format, not "select * from modules"

Reply:
yeah I for everything they seem to have devices.xxx